governance-exercises/Executive Vote 02.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Template - [Executive Vote] Enhanced Dai Savings Rate Activation, Spark Protocol Debt Ceiling Increase, Set Harbor Trade Debt Ceiling to Zero, AVC Members Rewards for Q2, DAO Resolution for Monetalis Clydesdale and Other Actions - August 2, 2023
+summary: Activate Enhanced Dai Savings Rate; increase Spark Protocol D3M Debt Ceiling; set Harbor Trade Credit Debt Ceiling to zero; AVC Members Rewards for Q2 2023; DAO Resolution for Monetalis Clydesdale JAT1 Surplus Return; Spark Protocol Proxy Spell Execution; distribute Launch Project funding; update New Silver RWA Agreement parameter.  
+date: 2023-08-02T00:00:00.000Z
+address: "$spell_address"
+
+
+
+---
+
+# [Executive Proposal] Enhanced Dai Savings Rate Activation, Spark Protocol Debt Ceiling Increase, Set Harbor Trade Debt Ceiling to Zero, AVC Members Rewards for Q2, DAO Resolution for Monetalis Clydesdale and Other Actions - August 2, 2023
+
+The Governance Facilitators, Dewiz, Sidestream and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following actions within the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The Enhanced Dai Savings Rate (EDSR) will be activated, increasing the DSR to 8%.
+- The Spark Protocol Dai Direct Deposit Module (D3M) Maximum Debt Ceiling will be increased to 200 million DAI. 
+- The Harbor Trade Credit vault's Debt Ceiling will be set to zero DAI, and the vault will be put into soft liquidation. 
+- AVC Members' Participation Rewards for Q2 will be distributed. 
+- A DAO Resolution pertaining to the Monetalis Clydesdale James Asset Trust 1 (JAT1) structure will be approved. 
+- A Spark Protocol Proxy Spell will be triggered to implement changes to the DAI interest rate strategy, WETH market parameters and DAI market parameters.
+- 2 million DAI of Launch Project funds will be distributed to a wallet controlled by the Accessibility Scope Facilitator for growth and marketing expenses.
```

Comment:
I feel like 'Launch Project' is a good place to link out to the relevant Atlas section.